package.json

```diff
@@ -50,6 +51,117 @@
     "chai": "^3.5.0",
     "vscode-textmate": "^2.1.1"
   },
+  "runtimeDependencies": [
```

Comment:
It looks like this pulls in both omnisharp-coreclr and omnisharp-desktop on windows. Is this the desired behavior?